### PR TITLE
Create a recommendation file for San Diego 

### LINKED ISSUE #11: Create a new file for githubteacher San Diego
- [x] Create a branch
- [x] Add File
- [x] Create a commit
- [x] Open a PR
- [x] Seek Approval
- [x] Merge PR

### PR DESCRIPTION
**Pull request by githubteacher**
closes #11

### CODE REVIEW

File: githubteacher-sandiego.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# San Diego, CA
```
Comment (githubteacher) [inline githubteacher-sandiego.md:1]: I need to add more detail!

Comment (geesusan) [inline githubteacher-sandiego.md:1]: A single comment.

**Comment by simmonsandy46** [comment]
:surfing_man: :house:

**Review by AudreyBramy — APPROVED** [review]
:+1:

**Pull request merged**